tests/test_term_call_modes.py

```diff
@@ -196,7 +196,9 @@ def test_term_call_modes(self):
                 domain_geometry = '%d_%s' % (domain.shape.dim, domain_geometry)
 
             for _, term_cls in ordered_iteritems(term_table):
-                if not domain_geometry in term_cls.geometries:
+                if (domain_geometry not in term_cls.geometries) \
+                   or ("dg" in term_cls.name) \
+                   or (term_cls.name == "dw_ns_dot_grad_s"):
```

Comment:
> I only now noticed that `tests/test_term_call_modes.py` fail for DG terms, this is beacause DG terms raise `ValueError` when someone tries to use them with field other than DGField.

I have looked into it and there are two issues

- one fundamental: the DG terms require boundary condition data. I don't think adding those to `test_term_call_modes.py` is a good idea so I omited DG terms  and will provide my own tests

- second a bit messy: `dw_ns_dot_grad_s`  implemented in terms_dg.py ([here](https://github.com/sfepy/sfepy/blob/6f2412096ee1ec1cfb73e7f29861e6a5706ac677/sfepy/terms/terms_dg.py#L677)) requires as parameters callables which the test can't provide. I dug a bit into it and do not see clean way to patch it, so I omit it from the test and will provide my own test too.